src/components/curation/Genotype.vue

```diff
@@ -0,0 +1,102 @@
+<script>
+import { convertStringToKebabCase } from "../../utility/CurationUtility";
+export default {
+  props: {
+    attributesData: Object,
+    allelicRequirement: String,
+    crossCuttingModifiers: Array,
+  },
+  emits: ["update:allelicRequirement", "update:crossCuttingModifiers"],
+  methods: {
+    checkboxHandler(crossCuttingModifier, checked) {
+      let updatedCrossCuttingModifiers = [...this.crossCuttingModifiers];
+      if (checked) {
+        updatedCrossCuttingModifiers.push(crossCuttingModifier);
+      } else {
+        updatedCrossCuttingModifiers.splice(
+          updatedCrossCuttingModifiers.indexOf(crossCuttingModifier),
+          1
+        );
+      }
+      this.$emit("update:crossCuttingModifiers", updatedCrossCuttingModifiers);
+    },
+    convertStringToKebabCase,
```

Comment:
Suggestion - Cross cutting modifier is the only one we do not display without "_", for uniformity, I think we either display all of them without "_" or display all of them the way they should be

Reply:
Ignore this comment.